docs/routing/introduction.md

```diff
@@ -74,6 +74,8 @@ In the example above we have multiple links, each one maps a path (`href`) to a
 - `/about` → `pages/about.js`
 - `/blog/hello-world` → `pages/blog/[slug].js`
 
+Any `<Link />` in the viewport (initially or through scroll) will be preloaded by default for pages using [Static Generation](/docs/basic-features/data-fetching.md#getstaticprops-static-generation). Server-rendered routes are _not_ preloaded.
```

Comment:
```suggestion
Any `<Link />` in the viewport (initially or through scroll) will be prefetched by default including the corresponding data for pages using [Static Generation](/docs/basic-features/data-fetching.md#getstaticprops-static-generation). The corresponding data for [server-rendered](https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering) routes is _not_ prefetched.
```

We should probably stress the data is not prefetched specifically as the needed page bundle is always prefetched by default. Also the router method is called `prefetch` so might be good to stick with that over preload.

Reply:
👍 I took the verbiage preloaded from the `next/link` docs. Should we change that, too?